charts/newrelic-infra-operator/templates/admission-webhooks/job-patch/psp.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if (and (not .Values.customTLSCertificate) (not .Values.certManager.enabled)) }}
+{{- if (and (not .Values.customTLSCertificate) (not .Values.certManager.enabled) (.Values.rbac.pspEnabled)) }}
```

Comment:
nit: we could add a unittest to cover this

Reply:
Done in 61abeb473578b14734476296b6a80de6b51b7fd1.